test/e2e/es/mutation_test.go

```diff
@@ -160,6 +160,21 @@ func TestMutationSecondMasterSetDown(t *testing.T) {
 	RunESMutation(t, b, mutated)
 }
 
+func TestMutationAndReversal(t *testing.T) {
```

Comment:
Maybe `TestConfigMutationAndReversal`?

Reply:
I find it convenient that I can run all mutation tests with `env TESTS_MATCH=TestMutation make e2e-local`

Reply:
👍 if we end up adding more mutation-reversal test (eg. change resource limits), we can rename it at this point.